### DISCUSSION TITLE
My first video solved and processed using OpenRadioss

### DISCUSSION THREAD

**Discussion by lcheng9** [OpenRadioss/OpenRadioss#939] [Show and tell]
Here is my first video solved and processed using OpenRadioss. Thank you all for making this open-source project real and answering my questions recently. 

https://youtu.be/L8fniIHB5Y0

**Comment by elequiniou** [comment]
That's great Icheng9 ! 
I heard recently about WelSim MatEditor supporting OpenRadioss...
Very good to see your first example solved with OpenRadioss! 
Thanks for your interest in this project!

**Reply by lcheng9** [reply]
Thank you @elequiniou. Yes, the OpenRadioss material scripts will be long-term supported in MatEditor. OpenRadioss solver is robust and accurate, I am pretty enjoying working on this topic right now and will try to make more videos recently.
